### PR TITLE
fix(Note): Expose `--light-text` CSS var, fixups

### PR DESCRIPTION
**Pull request by BYK**
`Note` component was using `--light-text` CSS var, which was not exposed here. This made `Note`s look identical with regular content. This patch exposes the variable, fixing the issue.

It also converts some notes to `Alert`s to get more attention and revises some grammar in them.

Old:

image

New:

image

### CODE REVIEW

**Comment by BYK** [comment]
Ping @PeloWriter

**Comment by dcramer** [comment]
Note is intentionally different than Alert, as its supposed to be less "in your face". Not sure what isn't working, but it's something thats needed in docs.

**Comment by BYK** [comment]
>Note is intentionally different than Alert, as its supposed to be less "in your face". Not sure what isn't working, but it's something thats needed in docs.

I mean, it was just looking like regular text and it was only used in 2 pages. Doesn't `<Alert level="info">` cover the same "less in your face" scenario while being more consistent?

**Comment by BYK** [comment]
@dcramer updated the PR summary with screenshots. I can remove the "Note" titles.

**Comment by dcramer** [comment]
Alert is a big blue box.

Note is supposed to be a diff font color. Maybe we're must missing the styles for some reason

**Comment by dcramer** [comment]
image

gray text here is note

**Comment by PeloWriter** [comment]
Note in our user docs is used extensively. Like so:
Screen Shot 2021-02-01 at 9 58 55 AM

**Comment by BYK** [comment]
>Note is supposed to be a diff font color. Maybe we're must missing the styles for some reason

Possibly, I'll look into that.

>gray text here is note

I don't find that very readable or eye-catching (maybe because I'm color-blind?) but sure I'll restore `<Note>` and try to get its styles in as replacing `<Note>` with `<Alert level="info">` seems like a larger discussion.

**Comment by BYK** [comment]
@dcramer @PeloWriter fixed, ready for another review.

File: src/docs/config.mdx [hunk 1]
```diff
@@ -113,9 +110,9 @@ LOGGING['default_level'] = 'WARNING'
 
 If logging in a particular module is not showing up when you expect it to, you should check the log level for that module in `src/sentry/conf/server.py` in the `LOGGING` variable.
 
-<Note>
-If you are a Sentry employee who is looking for logs in Kibana and they aren't showing up when you expect them to, one potential reason is that a field in the log message does not match Elasticsearch's schema. See the index patterns <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">here</a>.
-</Note>
+<Note><markdown>
```
Comment (PeloWriter) [inline src/docs/config.mdx:113]: I haven't found it necessary to use the <markdown></markdown> elements in user docs, FYI, but I don't think they'll cause issues.

File: src/docs/config.mdx [hunk 2]
```diff
@@ -113,9 +110,9 @@ LOGGING['default_level'] = 'WARNING'
...
+If you are a Sentry employee who is looking for logs in Kibana and they aren't showing up when you expect them to, one potential reason is that a field in the log message not matching Elasticsearch's schema. Check <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">the index patterns</a> to see whether this is the reason.
```
Comment (PeloWriter) [inline src/docs/config.mdx:114]:
```suggestion
If you are a Sentry employee looking for logs in Kibana and they aren't showing up when as expected, one potential reason is that a field in the log message does not match Elasticsearch's schema. Check <a href="https://kibana.getsentry.net/app/kibana#/management/kibana/indices/0ba261d0-97ca-11e9-90db-dfacda4da93b?_g=()&_a=(tab:indexedFields)">the index patterns</a> to see whether this is the reason.
```

File: src/docs/config.mdx [hunk 3]
```diff
@@ -269,9 +266,9 @@ The secret used to verify GitHub App Webhook event.
 
 Your app's private key.
 
-<Note>
-Make sure to replace newlines them with `\n` to preserve them.
-</Note>
+<Alert level="warning">
+Make sure to replace the newlines with `\n` to preserve them.
```
Comment (PeloWriter) [inline src/docs/config.mdx:270]:
```suggestion
Replace the newlines with `\n` to preserve them.
```

**Review by PeloWriter — APPROVED** [review]
You may not need the " markdown  /markdown" elements

File: src/docs/config.mdx (hunk 1 above)
Comment (BYK) [reply src/docs/config.mdx:113]: Eh, will leave them for now. May do another sweep to fix them.

**Pull request merged**